src/sys.rs

```diff
@@ -5,3 +5,27 @@
 #![allow(non_snake_case)]
 
 include!(concat!(env!("OUT_DIR"), "/bindings.rs"));
+
+pub unsafe fn can_frame_len(frame: *const can_frame) -> u8 {
+    #[cfg(feature = "can-dlc-unaliased")]
+    {
+        (*frame).can_dlc.len
```

Comment:
```suggestion
        (*frame).can_dlc
```

https://github.com/Hakuyume/socketcan-rs/runs/4034470729?check_suite_focus=true#step:6:191